src/Core/Components/List/ListComponentBase.razor.cs

```diff
@@ -364,8 +364,10 @@ protected override void OnParametersSet()
         {
             if (SelectedOption == null && Items != null && OptionSelected != null)
             {
-                TOption? item = Items.FirstOrDefault(i => OptionSelected.Invoke(i));
-                InternalValue = GetOptionValue(item);
+                var item = Items.FirstOrDefault(i => OptionSelected.Invoke(i));
+                var value = GetOptionValue(item);
+                InternalValue = value;
```

Comment:
Please note that this `ListComponentBase` class is used by all lists. A change here will affect all other lists, not just the **Combobox**. Have you checked that other components are not affected?

Reply:
I checked other components including Fluent Select, Fluent Autocomplete, Fluent Listbox. The examples on the demo page still work as before.